modules/ROOT/partials/misc/admon-available-docker-images.adoc

```diff
@@ -0,0 +1,4 @@
+[NOTE]
+====
+Currently, the Docker images are only supported on x86-64 (also known as AMD64) architecture processors.
+====
```

Comment:
If we want to use admon in partial, we should update the others to follow the same method.

```suggestion
[NOTE]
====
Currently, the Docker images are only supported on x86-64 (also known as AMD64) architecture processors.
====
```